lib/99 bottles.rb

```diff
@@ -0,0 +1,17 @@
+  def ninety_nine_bottles_of_beer
+    bottles = 100
+    bottles.downto(2) do |bottles|
+      if (bottles-1) > 1
+      puts (bottles-1).to_s + " Bottles of beer on the wall, " +
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Surrounding space missing for operator '-'.<br>Do not write to stdout. Use Rails' logger if you want to log.